src/RequestBuilder/ForgetRequestBuilder.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\RequestBuilder;
+
+use Fig\Http\Message\RequestMethodInterface;
+use Lmc\Matej\Exception\LogicException;
+use Lmc\Matej\Model\Assertion;
+use Lmc\Matej\Model\Command\ForgetUser;
+use Lmc\Matej\Model\Request;
+
+class ForgetRequestBuilder extends AbstractRequestBuilder
+{
+    protected const ENDPOINT_PATH = '/forget';
+
+    /** @var ForgetUser[] */
+    protected $users = [];
+
+    /** @return $this */
+    public function addUser(ForgetUser $user): self
+    {
+        $this->users[] = $user;
+
+        return $this;
+    }
+
+    /**
+     * @param ForgetUser[] $users
+     * @return $this
+     */
+    public function addUsers(array $users): self
+    {
+        foreach ($users as $user) {
+            $this->addUser($user);
+        }
+
+        return $this;
+    }
+
+    public function build(): Request
+    {
+        if (empty($this->users)) {
+            throw new LogicException('At least one user forget must be added to the builder before sending the request');
```

Comment:
I would follow messages from other builders and use the command name:

"At least one ForgetUser command must be added to the builder before sending the request"